excretor/src/models.rs

```diff
@@ -2,9 +2,9 @@ use sqlx::prelude::FromRow;
 
 #[derive(FromRow)]
 pub struct Message {
-    pub id: i32,
     pub channel_name: String,
     pub user_id: String,
+    #[sqlx(rename = "msg_text")]
     pub text: String,
     #[sqlx(rename = "ts")]
     pub timestamp: String,
```

Comment:
This isnt a string, but an SQL timestamp apparently see https://stackoverflow.com/a/61670747 and https://stackoverflow.com/a/75904277

Reply:
Done.